src/styles/header/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 50px;
+  background: $color-white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo-img {
+    margin-left: 50px;
+    margin-top: 10px;
+    margin-bottom: 6px;

```

Comment:
remove this, add left padding to header and try to align vertically without `margin`

Reply:
done